docs/guide/vec_envs.rst

```diff
@@ -5,19 +5,27 @@
 Vectorized Environments
 =======================
 
-Vectorized Environments are a way to multiprocess training. Instead of training a RL agent
-on 1 environment, it allows to train it on `n` environments using `n` processes.
-Because of that, `actions` passed to the environment are now a vector (of dimension `n`). It is the same for `observations`,
-`rewards` and end of episode signals (`dones`).
-
+Vectorized Environments are a method for multiprocess training. Instead of training an RL agent
+on 1 environment, it allows us to train it on `n` environments using `n` processes.
+Because of this, `actions` passed to the environment are now a vector (of dimension `n`).
+It is the same for `observations`, `rewards` and end of episode signals (`dones`).
+In the case of non-array observation spaces such as `Dict` or `Tuple`, where different sub-spaces
+may have different shapes, the sub-observations are vectors (of dimension `n`).
+
+============= ============== ======= ============ ======== ========= ================
+Name          Refactored     ``Box`` ``Discrete`` ``Dict`` ``Tuple`` Multi Processing
```

Comment:
Here I think you can remove the "Refactored" column. And I'm not sure about the "Multi Processing" one. @hill-a  ?

Reply:
"Refactored" made sense when the we were still converting the models to the Sklearn-like interface, I agree it is not needed anymore.

"Multi Processing" has been a tad annoying in my eyes, as I am not sure what it means, is it SubVecEnv compatible? MPI compatible? I still think we should keep it, as a lot of methods currently dont support multi environments acceleration, over multi CPU.